types/src/era_id.rs

```diff
@@ -59,6 +59,8 @@ impl EraId {
     ///
     /// For `u64::MAX`, this returns `u64::MAX` again: We want to make sure this doesn't panic, and
     /// that era number will never be reached in practice.
+    #[must_use]
+    #[allow(clippy::integer_arithmetic)] // The caller must make sure this doesn't overflow.
```

Comment:
On this branch we're using `saturating_add` inside, so this is unnecessary.

Reply:
Good catch! Fixed in https://github.com/casper-network/casper-node/pull/2638/commits/bf2d72e1a418a0f0016754553ab21bbe9ba40e6f